config/jobs/kubernetes-sigs/mcs-api/OWNERS

```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- JeremyOT
+- pmorie
+- lauralorenz
+- skitt
```

Comment:
FYI @JeremyOT @pmorie @lauralorenz @skitt

Reply:
If fully tenured citizens of MCS API are open, I'd nominate @nojnhuh for this list as well :)

Reply:
I’d approve that, although I can’t approve this PR ;-).

Thanks for taking care of this @nojnhuh, feel free to add yourself to the list.

Reply:
Done, thanks!

@ameukam @dims Could you PTAL?